docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -3286,7 +3286,8 @@ Return low-level information about the `exec` command `id`.
         "tty": true,
         "user": "1000"
       },
-      "Running": false
+      "Running": false,
+      "Pid": "42000"
```

Comment:
before I forget, or someone is "trigger happy" and merges; can you add this to the API changes for 1.25; https://github.com/docker/docker/blob/master/docs/reference/api/docker_remote_api.md#v125-api-changes